From 07fe28a624dbee0e1fb8129cfd8d907237d77c01 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Thu, 18 May 2017 14:50:03 +0800
Subject: [PATCH 0098/1239] arm64: dts: a3700: set lane2 comphy speed to 6Gbps

Lane2 is used for SATA, its phy speed is auto-negotiated at 6Gbps;
the lane2 phy speed in dts is synchronized to 6Gbps so that boot log
can print out correct lane2 phy speed.

Change-Id: I13aea4b218f1faf115aff5d42eede971544d66e4
Signed-off-by: Ken Ma <make@marvell.com>
---
 arch/arm/dts/armada-3720-db.dts          | 2 +-
 arch/arm/dts/armada-3720-espressobin.dts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/armada-3720-db.dts b/arch/arm/dts/armada-3720-db.dts
index 0f8b8c59b3..169828cbc4 100644
--- a/arch/arm/dts/armada-3720-db.dts
+++ b/arch/arm/dts/armada-3720-db.dts
@@ -112,7 +112,7 @@
 
 	phy2 {
 		phy-type = <COMPHY_TYPE_SATA0>;
-		phy-speed = <COMPHY_SPEED_5G>;
+		phy-speed = <COMPHY_SPEED_6G>;
 		phy-invert = <COMPHY_POLARITY_NO_INVERT>;
 	};
 };
diff --git a/arch/arm/dts/armada-3720-espressobin.dts b/arch/arm/dts/armada-3720-espressobin.dts
index c2bc2aac0e..f8fd4ed244 100644
--- a/arch/arm/dts/armada-3720-espressobin.dts
+++ b/arch/arm/dts/armada-3720-espressobin.dts
@@ -83,7 +83,7 @@
 
 	phy2 {
 		phy-type = <COMPHY_TYPE_SATA0>;
-		phy-speed = <COMPHY_SPEED_5G>;
+		phy-speed = <COMPHY_SPEED_6G>;
 	};
 };
 
-- 
2.29.0

